apps/site/assets/css/_tabbed-nav.scss

```diff
@@ -0,0 +1,169 @@
+.m-tabbed-nav {
+  @include make-container();
+  @include make-container-max-widths();
+  background-color: $brand-primary;
+  display: block;
+  margin-top: $space-32;
+  padding-left: 0;
+  padding-right: 0;
+
+  @include media-breakpoint-down(sm) {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: $space-24;
+  }
+
+  .m-tabbed-nav__items {
+    @include make-container-max-widths();
+    display: flex;
+    height: 89px;
+    margin-right: auto;
+    padding-left: $space-16;
+
+    @include media-breakpoint-down(md) {
+      height: 80px;
+      padding-left: $space-16;
+    }
+  
+    @include media-breakpoint-down(sm) {
+      height: 64px;
+      padding-left: $space-8;
+    }
+  
+    @include media-breakpoint-down(xs) {
+      height: 48px;
+      padding-left: $space-24;
+    }
+  }
+
+  .m-tabbed-nav__item-container {
+    display: flex;
+    margin-left: $space-32;
+    padding-left: $base-spacing;
+    padding-right: $base-spacing;
+
+    @include media-breakpoint-down(md) {
+      margin-left: $base-spacing;
+      padding-left: $base-spacing-sm;
+      padding-right: $base-spacing-sm;
+    }
+  
+    @include media-breakpoint-down(sm) {
+      margin-left: $base-spacing;
+      padding-left: $base-spacing-sm;
+      padding-right: $base-spacing-sm;
+    }
+  
+    @include media-breakpoint-down(xs) {
+      margin-left: $base-spacing-sm;
+      padding-left: $base-spacing-sm;
+      padding-right: $base-spacing-sm;
+    }
+
+    @include hover-focus-active {
+      color: $white;
+      text-decoration: none;
+      background-color: $brand-primary-darkest;
+
+      .m-tabbed-nav__item {
+        border-bottom: 8px solid $brand-primary-darkest;
+
+        @include media-breakpoint-down(sm) {
+          border-bottom: 4px solid $brand-primary-darkest;
+        }
+      }
+
+      .m-tabbed-nav__icon-text {
+        color: $white;
+        text-decoration: none;
+      }
+    }
+
+    @include media-breakpoint-down(md) {
+      height: 80px;
+    }
+  
+    @include media-breakpoint-down(sm) {
+      height: 64px;
+    }
+  
+    @include media-breakpoint-down(xs) {
+      height: 48px;
+    }
+  }
+
+  .m-tabbed-nav__item {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    font-weight: $font-weight-medium;
+    justify-content: center;
+    padding-bottom: $space-12;
```

Comment:
design nitpick: The text looks a bit too high up thanks to the spacing needed to accommodate the bottom border -- I think `calc(#{$space-12} - 8px)` (or 4px for the sm breakpoint) would work out well here.